share/completions/vmctl.fish

```diff
@@ -0,0 +1,4 @@
+
+complete -c vmctl -xa 'console create load log reload reset start status stop pause unpause send receive' -n 'not __fish_seen_subcommand_from list console create load log reload reset start status stop pause unpause send receive'
+complete -c vmctl -n '__fish_seen_subcommand_from console reload reset start status stop pause unpause send receive' -xa '( vmctl status | grep -v NAME | awk \'{print $NF}\' )'
```

Comment:
Preferably, this'd use `string`, since it's a builtin. The `grep` is simple - `string match -v`. I never really learned awk - what does the output look like and what are you trying to get out?

Reply:
```
vmctl status
   ID   PID VCPUS  MAXMEM  CURMEM     TTY        OWNER NAME
    1     -     1    512M       -       -         qbit ssb
    2     -     1    1.0G       -       -         qbit chrome
    3     -     1    512M       -       -         qbit alpine
    4     -     1    512G       -       -         qbit void
```

Reply:
So you want the last field there? Yeah, that's something that isn't quite as nice with `string`.

Try `vmctl status | string match -v 'NAME' | string replace -r '^(\s+\S+\s+){7}' ''`

(Obviously the "NAME" part isn't great because IIUC you could have a VM called "NAME-SOMETHING" - what you'd really want is something like `tail -n +2`, but we can probably live with that)

Reply:
This works if i do `string match -e -v`. Also I switched "NAME" to "MAXMEM" - because that's a super lame vm name ;D